test/contracts/bond/bond-factory.test.ts

```diff
@@ -152,62 +139,54 @@ describe('Bond Factory contract', () => {
                     }
                 ]
                 verifyBeneficiaryUpdateEvents(receipt, expectedEvents)
-                verifyBeneficiaryUpdateLogEvents(bonds, receipt, expectedEvents)
+                verifyBeneficiaryUpdateLogEvents(
+                    creator,
+                    receipt,
+                    expectedEvents
+                )
             })
 
-            it('only Super User', async () => {
+            it('only ownerr', async () => {
```

Comment:
nit: typo

Reply:
arrrrrgh! ...it be an ownerrr I say! 🏴‍☠️ 

_Yes, 'tis a typo_